src/set.rs

```diff
@@ -249,6 +257,39 @@ mod tests {
     const TEST_BATCH_SMALL: usize = 100;
     const TEST_REP: usize = 20;
 
+    #[test]
+    fn debug_when_empty() {
+        let set: ConSet<String> = ConSet::new();
+        assert_eq!("{}".to_owned(), format!("{:?}", set));
+    }
+
+    #[test]
+    fn debug_when_has_elements() {
+        let set: ConSet<&str> = ConSet::new();
+        assert!(set.insert("hello").is_none());
+        assert!(set.insert("world").is_none());
+        let expected = "{\"hello\", \"world\"}".to_owned();
+        let actual = format!("{:?}", set);
+
+        let mut expected_chars: Vec<char> = expected.chars().collect();
+        expected_chars.sort();
+        let mut actual_chars: Vec<char> = actual.chars().collect();
+        actual_chars.sort();
+        assert_eq!(expected_chars, actual_chars);
+    }
+
+    #[test]
+    fn debug_when_elements_are_added_and_removed() {
+        let set: ConSet<&str> = ConSet::new();
+        assert_eq!("{}".to_owned(), format!("{:?}", set));
+        assert!(set.insert("hello").is_none());
+        assert!(set.insert("hello").is_some());
+        assert!(set.insert("hello").is_some());
+        assert_eq!("{\"hello\"}".to_owned(), format!("{:?}", set));
+        assert!(set.remove("hello").is_some());
+        assert_eq!("{}".to_owned(), format!("{:?}", set));
```

Comment:
Few more instances of the above comparing…

Reply:
Too many inserts, a couple will be enough, I see.
But are you ok with the rest of the test? The idea was to test that the debug string is consistent with the property of the set (no duplicates, can be emptied)?

Reply:
I'm OK with the test and with the inserts, I was just referring to the `to_owned()`, same as with the `debug_when_empty` test :-). Just the same nitpick.

Reply:
Oh, yes, of course! `^_^`